vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/DateTimePickerIT.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2000-2019 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datetimepicker;
+
+import java.time.LocalDateTime;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+
+import com.vaadin.flow.component.datetimepicker.demo.DateTimePickerView;
+import com.vaadin.flow.component.datetimepicker.testbench.DateTimePickerElement;
+import com.vaadin.flow.testutil.AbstractComponentIT;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.testbench.TestBenchElement;
+
+/**
+ * Integration tests for the {@link DateTimePickerView}.
+ */
+@TestPath("date-time-picker-it")
+public class DateTimePickerIT extends AbstractComponentIT {
```

Comment:
Could have some more test cases:
- set value from server -> check the value-change-event message and that date-picker and time-picker individually have correct values
- select values from client manually, not only by setting the value-property, but by opening the overlays and clicking the options -> check value change message
- set locale -> date-picker and time-picker display values in correct format

Reply:
Tests for these have now been added and I also fixed some value propagation issues that were revealed. Web component has been updated to alpha2 now.